src/services/cleeng.account.service.ts

```diff
@@ -131,15 +133,37 @@ export async function getUser({ config }: { config: Config }) {
   };
 }
 
+interface CleengConsent {
+  broadcasterId: number;
+  enabledByDefault: boolean;
+  label: string;
+  name: string;
+  required: boolean;
+  value: string;
+  version: string;
+}
+
 export const getPublisherConsents: GetPublisherConsents = async (config) => {
   const { cleeng } = config.integrations;
   const response = await get(!!cleeng?.useSandbox, `/publishers/${cleeng?.id}/consents`);
 
   handleErrors(response.errors);
 
-  return {
-    consents: response?.responseData?.consents || [],
-  };
+  const consents = ((response?.responseData?.consents || []) as CleengConsent[]).map(
```

Comment:
Why do we need to map cleeng's response?

Reply:
For consistency between responses from different providers and wherever they're handled. The same kind of mapping is done in `inplayer.account.service` as well. For example, you'll notice that cleeng's initial value of a consent is constrained to a boolean value and is expressed in the property called `enabledByDefault`, while on the other hand an inplayer register field has `default_value`, so they're both mapped to `defaultValue`. There's also a boolean property called `isCustomRegisterField` that also has its usage.

Reply:
Yes but on InPlayer side we map all the responses in order to match the cleeng's ones, not the other way around.. This way we need to make sure nothing is broken on cleeng's side..